src/main/java/com/sergiofah/integracao/service/LineService.java

```diff
@@ -0,0 +1,18 @@
+package com.sergiofah.integracao.service;
+
+import com.sergiofah.integracao.model.LineDTO;
+
+import java.util.List;
+
+public class LineService {
+    public List<LineDTO> getLines() {
+        Service service = new Service();
```

Comment:
The  `service` attribute mus be a class global object, since class may will have more methods that would use the attribute.